src/backend/cuda/select.cpp

```diff
@@ -32,7 +35,7 @@ void select(Array<T> &out, const Array<char> &cond, const Array<T> &a,
 template<typename T, bool flip>
```

Comment:
would be nice to remove this template parameter as well.

Reply:
True, these are changes to backend API level and have to be done along with other backends. We can do this in later change where other backends also change the API to be consistent.